packages/babel-helper-module-transforms/src/normalize-and-load-metadata.js

```diff
@@ -171,7 +171,7 @@ function getModuleMetadata(
           if (reexport) {
             localData.delete(localName);
 
-            reexport.names.forEach(name => {
+            reexport.bindings.map(({ name }) => name).forEach(name => {
```

Comment:
Can you use `reexport.bindings.forEach({ name } => {` and avoid an unnecessary loop? Also on lines 186 and 200